qcengine/programs/nwchem/harvester.py

```diff
@@ -218,17 +218,18 @@ def harvest_outfile_pass(outtext):
                 psivar[f'{cc_plain} CORRELATION ENERGY'] = mobj.group(2)
                 psivar[f'{cc_plain} TOTAL ENERGY'] = mobj.group(3)
         #Other TCE
-        #mobj = re.findall(
-        #        r'^\s+' + r'Iterations converged' + r'\s*' +
-         #       r'^\s+' + r'(.*?)' + r' correlation energy / hartree' + r'\s+=\s*' + NUMBER + r'\s*'
-         #       r'^\s+' + r'(.*?)' + r' total energy / hartree' + r'\s+=\s*' + NUMBER + r'\s*$', 
-          #      outtext, re.MULTILINE | re. DOTALL)
+        for cc_name in [r'CCSD', r'CI', r'LCC']:
+            mobj = re.findall(
+                r'^\s+' + r'Iterations converged' + r'\s*' +
+                r'^\s+' + r'(.*?)' + r' correlation energy / hartree' + r'\s+=\s*' + NUMBER + r'\s*'
```

Comment:
I think the findall --> search, `(.*?)` --> `cc_name` patterns like mbpt and (t) above would be better here.